src/Day.js

```diff
@@ -136,7 +136,6 @@ export default class Day extends Component {
         className={className}
         tabIndex={tabIndex}
         style={style}
-        role="gridcell"
```

Comment:
@parkeragee do we need a role of `"button"` here if it's supposed to function as one?

Reply:
Adding a role of button throws a linting error stating that `aria-selected` is not supported by the role "button". So, we'd need to remove that `aria-selected` attribute. I think just removing the `gridcell` role would allow the screen reader to read the content while not introducing a breaking change (removing the aria-selected) attr.